reversestring-wca/prompts.md

```
@@ -0,0 +1,27 @@
+
+# Contexto
+
+Tengo un requerimiento que consiste en crear una página web (HTML con )lógica en JS (script.js) que me invierta el orden de una cadena de texto
+
+# Objetivo
+
+Quiero crear una una página web usando JS, que me permita:
+
+1.  Ingresar una cadena de texto en un input text.
+2.Hacer click en un botón llamado "Reverse"  
+   - Como salida me muestre un reverse string de la cadena de texto.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)